_activities/pyhep.md

```diff
@@ -65,6 +67,35 @@ with the aim to provide an environment to discuss and promote the usage of Pytho
 
 The advert and details on these workshops are given below.
 
+## PyHEP 2025 & PyHEP.dev 2025
+
+As in previous years, two events will be held again this year:
+
+PyHEP 2025, on October 27-30, is a continuation of the free, online workshop series, which has been bringing
+Python HEP developers and physicists together these past years.
+
+PyHEP.dev 2026 is intended for Python HEP software developers to plan a coherent roadmap and make priorities for the upcoming year. It will be held in-person in Seattle, WA (US) on July 14-17.
```

Comment:
```suggestion
PyHEP.dev 2025 is intended for Python HEP software developers and power users to plan a coherent roadmap and make priorities for the upcoming year. It will be held in-person in Seattle, WA (US) on July 14-17.
```